tests/src/Kernel/Paragraphs/LinksBlockTest.php

```diff
@@ -0,0 +1,350 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\Paragraphs;
+
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the "Links block" and "Social media follow" paragraphs.
+ */
+class LinksBlockTest extends ParagraphsTestBase {
+
+  /**
+   * Tests the rendering of the paragraph type.
+   */
+  public function testRendering(): void {
+    // Create social media follow paragraph with horizontal variant.
+    $paragraph = Paragraph::create([
+      'type' => 'oe_social_media_follow',
+      'field_oe_title' => 'Social media title',
+      'field_oe_social_media_variant' => 'horizontal',
+      'field_oe_links_block_background' => 'gray',
+      'field_oe_social_media_links' => $this->getSocialMediaLinks(),
+      'field_oe_social_media_see_more' => [
+        'title' => 'Other social networks',
+        'uri' => 'https://europa.eu/european-union/contact/social-networks_en',
+      ],
+    ]);
+    $paragraph->save();
+
+    // Testing: SocialMediaFollow horizontal gray.
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    $this->assertBackgroundGray($crawler);
```

Comment:
This is the best case to use pattern assertions. We can create the LinksBlock assertion class and move all the code in there. We can then use it for both the social and link blocks paragraph.